dbt/include/snowflake/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,18 @@
+{% macro snowflake__get_columns_spec_ddl() %}
+  {# loop through user_provided_columns to create DDL with data types and constraints #}
+  {% if config.get('constraints_enabled', False) %}
```

Comment:
This looks good, nice work!

Reply:
Move this if statement to `adapters.sql` similar to this: https://github.com/dbt-labs/dbt-core/pull/6271/files#diff-7246f19f4d12042d172ac61b3d0c4564ea27dd8825870b35425ede1ea41625dbR28